### ISSUE TITLE
Support a php-based environment file

### ISSUE THREAD

**Issue by lukehmu** [putyourlightson/craft-sherlock#21]
### Describe the feature
We have a few legacy sites which use the older style env (`.env.php` instead of `.env`).

On attempting to run a scan, we get the following error:
```
PHP Warning 'yii\base\ErrorException' with message 'fileperms(): stat failed for /home/dir/to/project/.env' 
...
```
We're assuming it's always expecting a `.env` file, so it's falling over. It'd be great to support the legacy `.env.php`.

### Versions
- Plugin version: 3.0.0
- Craft version: 3.5.14

**Comment by bencroker** [comment]
Thanks for reporting this. Fixed in https://github.com/putyourlightson/craft-sherlock/commit/aef5d3d5803a5d1baf0a86f419b5ed9aceca616f and added a file permission check for `.env.php` too for the next release.

**Comment by bencroker** [comment]
Released in version 3.1.0.
